mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -125,17 +118,18 @@ impl CommandHandler {
                 }
 
                 if subacks.is_empty() {
-                    return Ok(true);
+                    return Ok(());
                 }
             }
         }
 
-        Ok(false)
+        Err(CommandHandlerError::MissingSubacks(topics.join(", ")))
```

Comment:
shouldn't it be a list of missing topics? 
```suggestion
        Err(CommandHandlerError::MissingSubacks(subacks.join(", ")))
```

Reply:
if subacks will become a Vec, lets pass it inside an error

Reply:
oops. yes i agree